corehq/apps/sso/backends.py

```diff
@@ -84,6 +86,11 @@ def authenticate(self, request, username, idp_slug, is_handshake_successful):
         if not is_new_user and not web_user.is_active:
             web_user.is_active = True
             web_user.save()
+            log_user_change(by_domain=None, for_domain=None,
+                            by_domain_required_for_log=False, for_domain_required_for_log=False,
+                            couch_user=web_user, changed_by_user=web_user,
+                            fields_changed={'is_active': web_user.is_active},
+                            changed_via=USER_CHANGE_VIA_REACTIVATION)
```

Comment:
nit: The order to kwargs would be nice to retain as the calling function. I had added them in order of relevance which had a nice flow to it.

Reply:
Thank you. 5bbeaf408b5e83f6cb01af0d61bdf9aa85b84f3b